src/pages/en/guides/markdown-content.md

```diff
@@ -576,7 +576,16 @@ export default {
 };
 ```
 
-We also suggest [diving into their theme documentation](https://github.com/shikijs/shiki/blob/main/docs/themes.md#loading-theme) to explore loading custom theme, light vs dark mode toggles, or styling via CSS variables.
+#### Adding your own theme
+
+Instead of using a predefined theme from Shiki's module, you can provide a path to a custom theme saved in your public folder:
+
+```js title="astro.config.mjs"
+// Example 'ayu-dark' saved as public/custom-themes/ayudark.json
+theme: './custom-themes/ayu-dark.json',
+```
```

Comment:
Thanks for kicking this off @nujovich and thanks for investigating further @kevinzunigacuellar!

I think we can update this as follows. There’s no requirement for the imported file to be in a specific directory, so just showing it imported from the root directory in the example.

````suggestion
Instead of using one of Shiki’s predefined themes, you can import a custom theme from a local file.

```js title="astro.config.mjs"
import { defineConfig } from 'astro/config';
import customTheme from './my-shiki-theme.json';

export default defineConfig({
  markdown: {
    shikiConfig: { theme: customTheme },
  },
});
```
````

It is also possible to import a theme defined in a JS or TypeScript module, but as themes seem to be generally distributed as JSON, I think just showing a JSON example like this makes sense.

Reply:
I think this is a great solution! And I think that means this is now ready for docs, @nujovich!  🥳 

Thanks so much for this helpful contribution, and welcome to Team Docs as a first-time contributor!